pkg/framework/plugins/nodeutilization/nodeutilization.go

```diff
@@ -351,24 +356,8 @@ func evictPods(
 					}
 				}
 
-				keysAndValues := []interface{}{
-					"node", nodeInfo.node.Name,
-				}
-				if quantity, exists := nodeInfo.usage[v1.ResourceCPU]; exists {
-					keysAndValues = append(keysAndValues, "CPU", quantity.MilliValue())
-				}
-				if quantity, exists := nodeInfo.usage[v1.ResourceMemory]; exists {
-					keysAndValues = append(keysAndValues, "Mem", quantity.Value())
-				}
-				if quantity, exists := nodeInfo.usage[v1.ResourcePods]; exists {
-					keysAndValues = append(keysAndValues, "Pods", quantity.Value())
-				}
-				for name := range totalAvailableUsage {
```

Comment:
so the mixing of node and total stats was incorrect

Reply:
Yeah. I probably misunderstood your previous comment.